src/openfermion/measurements/_bayesian_estimators.py

```diff
@@ -0,0 +1,774 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+'''
+Classes for estimators for quantum phase estimation. Estimators described
+in arXiv:XXXX.XXXX
+'''
+
+import numpy as np
```

Comment:
As annoying as this is, our style guide does not allow "import _ as _". Please just write out numpy.

Reply:
Done